docs/tutorials/troubleshooting/deploy-vm.md

```diff
@@ -0,0 +1,54 @@
+# Deploy VM
+This page is dedicated to diagnose issues that can arise during the deployment of virtual machines (VMs). Refer to [Deploy vm](https://vm-operator.readthedocs.io/en/stable/tutorials/deploy-vm/) on how to deploy a virtual machine.
+
+## Troubleshoot step
+### Describe Virtual Machine k8s resource
+After you ssh into the supervisor cluster control plane VM, run `kubectl describe vm <vm-name> -n <namespace-name>` command. Check `VM.Status.Conditions` as seen in sample output below.
```

Comment:
I don't think it's common for users to connect to Supervisor via SSH. I would use phrases like "after connecting to the Supervisor cluster" or "after accessing your namespace in the Kubernetes environment" instead.